website/content/docs/platform/mssql/upgrading.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+page_title: Upgrade the Vault EKM Provider
+description: Upgrade steps for the Vault EKM Provider for Microsoft SQL Server.
+---
+
+# Upgrading the Vault EKM Provider
+
+-> **Note:** If you get a "Files in Use" prompt while running the installer, select
+    "Do not close applications. A reboot will be required".
+
+~> **Note:** The upgrade process will put the database into maintenance mode and
+    require a restart. It is highly recommended to test this procedure in a
+    staging environment prior to running it on a production database.
+
+1. Disable the Vault EKM Provider, and verify it is marked disabled:
+
+    ```sql
+    ALTER CRYPTOGRAPHIC PROVIDER TransitVaultProvider DISABLE;
+    SELECT * FROM sys.cryptographic_providers;
+    ```
+
+1. Restart the Microsoft SQL Server service.
+1. Install the latest version from [releases.hashicorp.com][releases], following
+   the instructions in the installer.
+1. Reload the provider DLL:
+
+    ```sql
+    ALTER CRYPTOGRAPHIC PROVIDER TransitVaultProvider
+        FROM FILE = 'C:\Program Files\HashiCorp\Transit Vault EKM Provider\TransitVaultEKM.dll'
```

Comment:
I guess because this is a single statement it doesn't need a `;` at the end?

Reply:
Oh good spot. Yeah it's not required for a single statement, but I think it would be nice for consistency to include it.